src/components/src/test/java/org/apache/jmeter/gui/action/TestSave.java

```diff
@@ -28,25 +28,24 @@
 import org.junit.Test;
 
 public class TestSave {
-        private Save save;
+    private Save save;
 
+    @BeforeEach
```

Comment:
Is this a bit of JUnit5?

Reply:
Yeah, will fix that, thank you for spotting it.